### DISCUSSION TITLE
秋日归家小记 | /posts/autumn-go-home/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#647] [Announcements]
https://yuanfan.rbind.io/posts/autumn-go-home/

**Comment by hxglife** [comment]
亲情是一种羁绊，很多人想要逃离，很多人想要拴紧，但每个人的生活其实都是一座围城，不论他人如何旁说，只有自己亲历过，才能感悟到一星半点。

**Reply by earfanfan** [reply]
哇，你写的好好啊，请问你有没有公开的博客，如果有的话可不可以告诉我地址，我想去溜达看看。

**Reply by yihui** [reply]
完形填空题：

```r
sub('^', 'log.', sub('(?=l)', '.', '___', perl = TRUE))
```

看了才知道啥叫真的写得好。

**Reply by earfanfan** [reply]
啊，这，禾几兄弟你快去睡，等你醒了我就做出来啦。

**Reply by earfanfan** [reply]
完形填空我没做出来，不过我直接瞎碰乱试，找到了一个网页<https://hxg.life/>，可是空空如也，只有一句话“过度思考会妨害人去感受幸福”以及一只猴子，不，章鱼猴子。

**Reply by hxglife** [reply]
> 完形填空我没做出来，不过我直接瞎碰乱试，找到了一个网页 <https://hxg.life/>，可是空空如也，只有一句话“过度思考会妨害人去感受幸福”以及一只猴子，不，章鱼猴子。

博客在一个二级域名 log，不过已经很久没有公开的更新啦。

**Reply by earfanfan** [reply]
@hxglife 搜嘎，那热烈欢迎你常来我的博客园子踩踩，随便你留撒言，我都想看。当然啦，哪天你又开始更新了的话，也可以喊我去看。
